cmd/admin-router.go

```diff
@@ -188,6 +188,14 @@ func registerAdminRouter(router *mux.Router, enableConfigOps bool) {
 		adminRouter.Methods(http.MethodDelete).Path(adminVersion+"/remove-remote-target").HandlerFunc(
 			gz(httpTraceHdrs(adminAPI.RemoveRemoteTargetHandler))).Queries("bucket", "{bucket:.*}", "arn", "{arn:.*}")
 
+		// Bucket migration operations
+		// ExportBucketMetaHandler
+		adminRouter.Methods(http.MethodGet).Path(adminVersion+"/export-bucket-metadata").HandlerFunc(
```

Comment:
```suggestion
		adminRouter.Methods(http.MethodGet).Path(adminVersion+"/export-bucket-metadata").HandlerFunc(
```

If you wish to keep bucket optional, remove the bucket query param.